### PR TITLE
Add some tests for extension telemetry

### PR DESCRIPTION
**Pull request by lramos15**
Adds some tests for extension telemetry API

### CODE REVIEW

File: src/vs/platform/telemetry/test/common/telemetryLogAppender.test.ts
Comment (lramos15) [inline src/vs/platform/telemetry/test/common/telemetryLogAppender.test.ts:61]: @sandy081 Is there a better just generalized test logger I could use? I felt weird stealing this one but it is a telemetry test for the extension host.

File: src/vs/workbench/api/test/browser/extHostTelemetry.test.ts
Comment (lramos15) [inline src/vs/workbench/api/test/browser/extHostTelemetry.test.ts:20]: Is this the suggested way to do things @jrieken? Just create these bogus environment objects and Extension identifiers? I don't need most of these properties, but this seemed like the correct way to do it. Let me know if there's a better way to do things :)

**Review by sbatten — DISMISSED** [review]
looks like there are pending questions, but approving if you want to adjust after merge

File: src/vs/workbench/api/test/browser/extHostTelemetry.test.ts
Comment (jrieken) [reply src/vs/workbench/api/test/browser/extHostTelemetry.test.ts:20]: Yeah, I like to mock/fake as much as possible so that my dependencies stay small. Also note that we have the [mock](https://github.com/microsoft/vscode/blob/4674509f002f98319f48e44460561c85b3d8ad15/src/vs/base/test/common/mock.ts#L12)-util which makes it super easy to mock and/or partially implement any service or dependency

**Review by jrieken — DISMISSED** [review]

**Review by jrieken — APPROVED** [review]

File: src/vs/platform/telemetry/test/common/telemetryLogAppender.test.ts
Comment (sandy081) [reply src/vs/platform/telemetry/test/common/telemetryLogAppender.test.ts:61]: You can use [NullLoggerService](https://github.com/microsoft/vscode/blob/c6fa8b7e03da3071f0b90ca2a1c07844c9eab470/src/vs/platform/log/common/log.ts#L610)

**Pull request merged**
